django/db/migrations/operations/models.py

```diff
@@ -483,6 +483,28 @@ def reduce(self, operation, app_label=None):
         return super().reduce(operation, app_label=app_label)
 
 
+class AlterModelBases(ModelOperation):
+    reduce_to_sql = False
+    reversible = True
+
+    def __init__(self, model_name, bases):
+        self.model_name = model_name
+        self.bases = bases
```

Comment:
Should we add sanity-check for duplicates in `bases`, `_check_for_duplicates()`? (e.g. [`CreateModel.__init__()`](https://github.com/django/django/blob/ef082ebb84f00e38af4e8880d04e8365c2766d34/django/db/migrations/operations/models.py#L55-L59)).

Reply:
`name` should be used to follow `ModelOperation.__init__` signature and `super().__init__(name)` should be called.